app/request_lookup.py

```diff
@@ -46,26 +78,69 @@ def get_request_json(data_type, search_value):
         }
     ]
 
+    if language != 'en':
+        should_terms.append({
+            "term":
+                {
+                        language:
+                            {
+                                "value": search_value,
+                                "boost": 3.0
+                            }
+                    }
+
+        })
+
     # Then weight on each individual word
-    if len(search_value.split()) > 1:
-        for word in search_value.split():
+    split_search_values = search_value.split()
+    if len(split_search_values) > 1:
+        for word in split_search_values:
             should_terms.append({
                 "term":
                 {
-                    data_type:
+                    "en":
                         {
                             "value": word,
                             "boost": 2.0
                         }
                 }
             })
 
+        if language != 'en':
+            for word in split_search_values:
+                should_terms.append({
```

Comment:
We could remove this for loop and move this append to the first loop with just the `if`?